client/src/components/application/Examine/RequestInfoHeader.vue

```diff
@@ -144,6 +144,9 @@
             <v-flex :class="!consumptionDate ? 'grey--text' : ''">
               <b>Consumed:</b> {{ consumptionDate ? consumptionDate : 'n/a' }}
             </v-flex>
+            <v-flex :class="!consumedBy ? 'grey--text' : ''">
+              <b>Consumed By:</b> {{ consumedBy ? consumedBy : 'n/a' }}
```

Comment:
Would `consumedBy || 'n/a'` work here?

Reply:
I'll give it a try!

Same as above (`consumptionDate`)?

Reply:
Sure, if it works. (I think it will.) It's a tiny optimization since the variable is only dereferenced once.

Reply:
Works! Updated in next commit